tests/test-client/main.cpp

```diff
@@ -60,6 +60,42 @@ struct CurlGlobal
     }
 };
 
+class CustomLogTarget : public easyloggingpp::ILogTarget
+{
+public:
+    CustomLogTarget()
+        : count_(0)
+    {
+    }
+
+    void log(const std::string& msg) {
+        ++count_;
+        std::cout << "log(" << count_ << "): " << msg << std::endl;
+    }
+
+private:
+    int count_;
+};
+
+// TODO: move to easyloggingpp code
+class ScopedLogTarget
```

Comment:
This class is not a log target. Maybe It should be called something like `ScopedLogTargetGuard`?